wherehows-kafka/src/main/java/wherehows/processors/MetadataLineageProcessor.java

```diff
@@ -40,14 +42,15 @@
 
   private final String whitelistStr = config.hasPath("whitelist.mle") ? config.getString("whitelist.mle") : "";
```

Comment:
Should be able to do cofig.getString("whitelist.mle", "")?

Reply:
Didn't find such function with default value

Reply:
I see.